Config/Config-debug.xcconfig

```diff
@@ -0,0 +1,4 @@
+#include "../Pods/Target Support Files/Pods-MiniApp_Example/Pods-MiniApp_Example.debug.xcconfig"
```

Comment:
The xcconfig files are specifically for the example app so should they be in the `Example/` folder?

Reply:
that makes sense